css/MLS.css

```diff
@@ -89,3 +89,6 @@ a:hover { text-decoration: underline; }
 .ltx_indexrefs > .ltx_text:first-child:before {
   content: " –";
 }
+
+.ltx_page_header *[rel~="up"],
+.ltx_page_footer *[rel~="up"]   { display:table; margin: 0 auto; text-align:center; }
```

Comment:
Matching formatting with surrounding code:
```suggestion
.ltx_page_footer *[rel~="up"] { display: table; margin: 0 auto; text-align: center; }
```

Reply:
Looks good now.  Resolving.